scripts/gWMS-CMSRunAnalysis.sh

```diff
@@ -246,7 +175,7 @@ echo "======== python bootstrap for stageout at $(TZ=GMT date) FINISHED ========
 echo "======== Stageout at $(TZ=GMT date) STARTING ========"
 rm -f wmcore_initialized
 # Note we prevent buffering of stdout/err -- this is due to observed issues in mixing of out/err for stageout plugins
-PYTHONUNBUFFERED=1 python2.7 cmscp.py
+PYTHONUNBUFFERED=1 $pythonCommand cmscp.py
 STAGEOUT_EXIT_STATUS=$?
 
 if [ ! -e wmcore_initialized ];
```

Comment:
is this a good time to review what this thing really is and whether we need it ? There should be a better way to flag that imports from WMCore fail in our python scripts ! Also: message in Line 183 can be made more clear

Reply:
addressed in this comment: https://github.com/dmwm/CRABServer/pull/7441#issuecomment-1326764235